docs/_tutorials/confirmed-delivery.md

```diff
@@ -12,43 +15,34 @@ This tutorial builds on the basic concepts introduced in [Persistence with MQTT]
 This tutorial assumes the following:
 
 *   You are familiar with Solace [core concepts]({{ site.docs-core-concepts }}){:target="_top"}.
-*   You have access to a running Solace message router with the following configuration:
-    *   Enabled message VPN configured for guaranteed messaging support.
-    *   Enabled client username
+*   You have access to Solace messaging with the following configuration:
+    *   Connectivity information for a Solace message-VPN configured for guaranteed messaging support
+    *   Enabled client username and password
     *   Client-profile enabled with guaranteed messaging permissions.
-    *   Enabled MQTT service on port 1883
+    *   Enabled MQTT service
 
-One simple way to get access to a Solace message router is to start a Solace VMR load as [outlined here]({{ site.docs-vmr-setup }}){:target="_top"}. By default the Solace VMR will run with the “default” message VPN configured and ready for messaging and the MQTT service enabled on port 1883\. Going forward, this tutorial assumes that you are using the Solace VMR. If you are using a different Solace message router configuration, adapt the instructions to match your configuration.
+{% if jekyll.environment == 'solaceCloud' %}
+One simple way to get access to Solace messaging quickly is to create a messaging service in Solace Cloud [as outlined here]({{ site.links-solaceCloud-setup}}){:target="_top"}. You can find other ways to get access to Solace messaging on the [home page]({{ site.baseurl }}/) of these tutorials.
```

Comment:
@macstewart, I just noticed an issue with this boiler plate line. We did not give more way to access solace messaging on the home page. After review we in fact give more ways to access Solace messaging in the solaceMessaging.md section.

Reply:
So perhaps say "You can find other ways to get access to Solace messaging below". Maybe that' sufficient?